### DISCUSSION TITLE
FreeFormPlane Geometry Merge

### DISCUSSION THREAD

**Discussion by andrejhronco** [cables-gl/cables_docs#120] [Answers & Questions]
Hi, I was wondering how I could create unequal sided polygons in cables. I tried using two FreeFormPlane ops and combine them with GeometryMerge, but that doesn't seem to create a single geometry. when I try the same with two Triangle ops it creates a single geometry. Also, it seems that combining FreeFormPlane + GeometryMerge really slows the renderer down.

**Comment by tastez** [comment]
hi @AndrejHronco  - you could also approach this with Triangulate2DPath and generate some polys that way. You do end up with a 2D plane poly made of triangles, but this might be what you are looking for. Of course you can manipulate the shape further with some tricks.

https://cables.gl/p/C5taZV

**Reply by andrejhronco** [reply]
Thanks, Interesting Op. I don't think it's what I'm looking for though. I want to create an oblong hexagon and want control over width of each side. FreeFormPlane is a more intuitive Op to use than Triangulate2DPath. Is there another way to be able to create modified planes?
